Cell.cs

```diff
@@ -0,0 +1,74 @@
+enum FireResult {
+  Missed, 
+  Hit,
+  Killed,
+  Duplicate
+}
+
+enum CellStatus {
+  Hidden,
+  ShipHit,
+  Open
+}
+
+interface ICell
+{
+  public FireResult Fire(Coord coord);
+}
+
+class Cell 
+{
+  Coord coord;
```

Comment:
Use explicit access modifiers for fields as a good practice. Use readonly if applicable for a field. Use `_` prefix for fields